api/src/opentrons/protocols/parameters/validation.py

```diff
@@ -192,21 +208,25 @@ def _validate_min_and_max(
             "If a maximum value is provided a minimum must also be provided."
         )
     elif maximum is not None and minimum is not None:
-        if isinstance(maximum, (int, float)) and isinstance(minimum, (int, float)):
-            if maximum <= minimum:
+        if parameter_type is int or parameter_type is float:
+            if not isinstance(minimum, parameter_type):
                 raise ParameterDefinitionError(
-                    "Maximum must be greater than the minimum"
+                    f"Minimum is type {type(minimum)}, must match parameter type {parameter_type}"
```

Comment:
comma splice, easy to fix by adding a conjunction
```suggestion
                    f"Minimum is type {type(minimum)}, but must match parameter type {parameter_type}"
```
replace `match` here if changed above.